Jenkinsfile

```diff
@@ -6,11 +6,12 @@ pipeline {
         buildDiscarder(logRotator(numToKeepStr:'25'))
     }
     triggers {
-        cron('H H(20-23) * * *')
+        cron('H H(17-19) * * *')
     }
     environment {
         DOCS = 'distribution/docs'
         ITESTS= 'distribution/test/itests/test-itests-ddf'
+        JDK_LOC = 'C:/Program Files/Java/jdk1.8.0_131'
```

Comment:
We should be able to just use the `jdk` property in the `withMaven` step (this is currently being verified in another pipeline at the moment, but i'm pretty sure it already got us past the `JAVA_HOME` issue). This will enable us to either make a project follow the latest jdk8 version/update or pin the project to a specific version if necessary